src/q6/mecasols-GCIV1072/summary/mecasols-GCIV1072-summary.tex

```diff
@@ -0,0 +1,12 @@
+\documentclass[fr]{../../../eplsummary}
```

Comment:
en pas fr ici non ?

Reply:
Toute la synthèse est écrite en français, mais c'est vrai que cela aurait été mieux de l'avoir en anglais puisque le cours se donne sans doute en anglais.

Reply:
En fait y a des parties en anglais et certaines en français. Je suis tombé sur une partie en anglais (e.g. "Fundamental requirement for stability of slope...") et j'ai cru que tout était en anglais ^^

Reply:
Ah oui, bon on va devoir rajouter une option "franglais" pour des cas ainsi :smile:  .